_posts/2016-09-01-500-issues.md

```diff
@@ -4,11 +4,40 @@ title:  v5.0.0 Known Issues
 tags:
 - Known Issues
 ---
-###### Google Chrome Known Issues:
 
-[RES does not work in Chrome 53 and 54, but canary builds are fine (Chrome bug, awaiting fix release)](https://github.com/honestbleeps/Reddit-Enhancement-Suite/issues/3342)
+Listing of known issues: https://github.com/honestbleeps/Reddit-Enhancement-Suite/milestone/19
 
-[Never Ending Reddit sometimes fails to work (We are investigating this)](https://github.com/honestbleeps/Reddit-Enhancement-Suite/issues/3341)
+###### RES broken in Chrome with JavaScript Harmony enabled
+
+RES is not compatible with chrome's experimental JavaScript harmony functionality.
+
+Solution: turn off chrome://flags/#enable-javascript-harmony
+
+More details: https://github.com/honestbleeps/Reddit-Enhancement-Suite/issues/3344
+
+----
+
+###### RES broken sometimes in Chrome 53/54
+
+RES does not work in Chrome 53 and 54 because of a Chrome issue, but Chrome 55 builds are fine. Hopefully Chrome will push a fix.
```

Comment:
Actually, it seems it does work fine in 53 as long as you don't have the `enable-javascript-harmony` flag enabled. 54 is broken with or without the flag, and 55 works with or without the flag.